src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+package mate.jdbc.dao;
+
+import static mate.jdbc.util.SqlQueries.DELETE_MANUFACTURER_BY_ID;
+import static mate.jdbc.util.SqlQueries.INSERT_MANUFACTURER;
+import static mate.jdbc.util.SqlQueries.SELECT_ALL_MANUFACTURERS;
+import static mate.jdbc.util.SqlQueries.SELECT_MANUFACTURER_BY_ID;
+import static mate.jdbc.util.SqlQueries.UPDATE_MANUFACTURER;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private final Connection connection = ConnectionUtil.getConnection();
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (PreparedStatement createManufacturerStatement = connection.prepareStatement(
+                INSERT_MANUFACTURER, PreparedStatement.RETURN_GENERATED_KEYS)) {
```

Comment:
```suggestion
        String insertManufacturer = "";
        try (Connection connection = ConnectionUtil.getConnection();
                PreparedStatement createManufacturerStatement =
                        connection.prepareStatement(insertManufacturer,
                                Statement.RETURN_GENERATED_KEYS)) {
```
remove static imports of queries
and write them here